client-v2/src/lib/dnd/DropZone.tsx

```diff
@@ -37,10 +43,19 @@ class DropZone extends React.Component<Props, State> {
   }
 
   public render() {
-    return this.props.children(this.state.isTarget);
+    return (
+      <DropZoneContainer isActive={this.state.isActive}>
+        {this.props.children(this.state.isTarget)}
+      </DropZoneContainer>
+    );
   }
 
   private handleStoreUpdate: Sub = (id, hoverIndex) => {
+    const state = this.props.store && this.props.store.getState();
```

Comment:
This is definitely the right way to do this now put potentially some form of PureComponent wrapper might be a bit nicer here in future given we’re just checking for changes everywhere.

Reply:
👍